<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeTypesPage</class>
 <widget class="QGroupBox" name="NodeTypesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Node Types</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="l_nodeType">
     <property name="text">
      <string>Node type name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_nodeType"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="b_add">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="list_nodeTypes">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
